app/models/event.rb

```diff
@@ -84,6 +85,10 @@ def registration_possible?
     registrations.count < max_attendees
   end
 
+  def ended?
+    Time.now.to_i > event_schedules.find_by(schedule_id: program.selected_schedule_id).end_time.to_i
```

Comment:
I think you should use the timezone of the conference this event belongs to. And you can compare times directly without converting them to string.

``` Ruby
timezone = program.conference.timezone
Time.find_zone(timezone).now > event_schedules.find_by(schedule: program.selected_schedule).end_time
```